estimate.go

```diff
@@ -12,6 +12,8 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/logrusorgru/aurora"
```

Comment:
this library is not packaged in debian too. it's not so important to have colorful output...